db/migrate/20200505201706_create_demux_transmissions.rb

```diff
@@ -10,6 +10,7 @@ def change
       t.string :response_code
       t.jsonb :response_headers
       t.text :response_body
+      t.text :context
```

Comment:
Should we just use jsonb here? Per discussion with Josh

Reply:
I'm going to go ahead and make this change. The logic behind not using a jsonb type was to try and prevent being database specific since jsonb is a postgres data type. But, as Josh pointed out we are already using jsonb for the response/ request headers (in that case, to make it easier to search for things in the given headers).

Because of that, we should either remove all jsonb and use serialization into a text field instead; or we should probably stay consistent and use jsonb.

I'm favoring just using jsonb like we did with the headers, and then if we ever want to make this database agnostic this is one of the things that will have to be accounted for.